tests/check_non_terminating.sh

```diff
@@ -107,6 +107,13 @@ isNonTerminating() {
   if kubectl wait pods -n "${TEST_NAMESPACE}" test-terminating --for condition=Ready --timeout=${timeout_in_sec}s >/dev/null 2>&1; then
     echo "  SUCCESS: The container started successfully and didn't terminate"
     kubectl delete pod test-terminating -n "${TEST_NAMESPACE}" >/dev/null 2>&1
+
+    # remove image to save space
+    if [ "$ENV" = "minikube" ]; then
+      echo "  COMMAND: \"minikube ssh docker image rm ${image}\""
```

Comment:
Maybe is better to use int_image instead of image? Only because we use it like this in this func